mysk-data-api/src/routes/v1/clubs/get_club_statistics.rs

```diff
@@ -0,0 +1,73 @@
+use crate::AppState;
+use actix_web::{get, web::Data, HttpResponse, Responder};
+use mysk_lib::{
+    common::response::ResponseType, helpers::date::get_current_academic_year, prelude::*,
+};
+use serde::Serialize;
+use sqlx::{query, PgPool};
+
+#[derive(Debug, Serialize)]
+struct ClubStatistics {
+    pub new_club_members: i64,
```

Comment:
This might be a bit petty, but I think you can remove the word “new”.

Reply:
naming is hard but I changed it to
- `club_members`
- `club_staffs`
instead